### PR TITLE
Add blended model (USE THIS ONE)

### PR DESCRIPTION
**Pull request by cjauvin**
This is a version of https://github.com/CSHS-CWRA/RavenPy/pull/72 where I just branched off 0.3.0 and cherry picked the relevant commits, to avoid any confusion.

### CODE REVIEW

**Review by Zeitsperre — APPROVED** [review]
LGTM, but I don't know what most of this code does, haha

File: ravenpy/models/raven-blended/raven-blended.rvh [hunk 1]
```diff
@@ -0,0 +1,16 @@
...
+:CreationDate      Mar 2021
+#
+# Emulation of BLENDED simulation of Salmon River near Prince George
+#------------------------------------------------------------------------
+#
+#
+:SubBasins
+        :Attributes     NAME    DOWNSTREAM_ID   PROFILE   REACH_LENGTH    GAUGED
+        :Units          none    none            none      km              none
+        1,             {name},   -1,             NONE,     _AUTO,          1
```
Comment (julemai) [inline ravenpy/models/raven-blended/raven-blended.rvh:13]: Does get filled in with "None"

**Comment by julemai** [comment]
Committed a few adjustments in documentation. The rest looks fine to me. Only the model name in RVH does not get filled in properly. But tests are passing.

**Comment by huard** [comment]
I suggest we revisit the `name` issue in our "semi-distributed config" work.

**Pull request merged**